docs/src/lib/binary_functions.md

```diff
@@ -49,6 +49,12 @@ convex_hull(::VPolytope{N}, ::VPolytope{N}) where {N<:Real}
 convex_hull(::VPolygon{N}, ::VPolygon{N}) where {N<:Real}
 ```
 
+## Minkowski sum
+
+```@docs
+minkowski_sum(::AbstractPolytope{N}, ::AbstractPolytope{N}) where {N<:Real}
```

Comment:
Hm, I do not like to have two places in the manual for this. Let us put it in `interfaces.md` only and then move all `minkowski_sum`s to this file in another PR.